bigtable-dataflow-parent/bigtable-beam-import/README.md

```diff
@@ -11,7 +11,7 @@ you can build them using Maven.
 
 [//]: # ({x-version-update-start:bigtable-dataflow-parent:released})
 ### Download the jars
-Download [the import/export jars](http://search.maven.org/remotecontent?filepath=com/google/cloud/bigtable/bigtable-beam-import/1.20.0/bigtable-beam-import-1.20.0-shaded.jar), which is an aggregation of all required jars.
+Download [the import/export jars](https://search.maven.org/search?q=a:bigtable-beam-import), which is an aggregation of all required jars.
```

Comment:
just to make this super explicit, I would also add the group id (`com.google.cloud.bigtable`) to the search

Reply:
Or better yet: https://search.maven.org/artifact/com.google.cloud.bigtable/bigtable-beam-import

Reply:
updated